utils/verify-specs.js

```
@@ -0,0 +1,39 @@
+const fs = require('fs/promises');
```

See my comments on the package.json file.  I really don't know if we need this file since a test shouldn't be added, and neither can run without having a .spec file in the features directory.  No tests should even be added/merged into the main branch without it. Plus in the future, every team will have its own repositories which would then have their .spec files located there and not in Nala's repo.  I guess I'm just trying to understand the need for this validation checker.

---

Is this for the UI, to ensure the titles for the reporting are correct and have what is needed?  If so, how are these going to be connected, and what is the process for the usage of this validator?

---

my original intent was to catch errors fail early and not during test runs. But this is definitely not necessary, so I'll remove it